src/pages/en/editor-setup.md

```diff
@@ -28,7 +28,8 @@ To get started, install the [Astro VS Code Extension](https://marketplace.visual
 Our amazing community maintains several extensions for other popular editors, including:
 
 - [VS Code Extension on Open VSX](https://open-vsx.org/extension/astro-build/astro-vscode) <span style="margin: 0.25em;"><Badge variant="accent">Official</Badge></span> - The official Astro VS Code Extension, available on the Open VSX registry for open platforms like [VSCodium](https://vscodium.com/)
-- [Nova Extension](https://extensions.panic.com/extensions/sciencefidelity/sciencefidelity.astro/)<span style="margin: 0.25em;"><Badge variant="neutral">Community</Badge></span> - Syntax highlighting, IntelliSense, autocompletion for Astro
+- [Nova Extension](https://extensions.panic.com/extensions/sciencefidelity/sciencefidelity.astro/) <span style="margin: 0.25em;"><Badge variant="neutral">Community</Badge></span> - Syntax highlighting, IntelliSense, autocompletion for Astro
+- Neovim [LSP](https://github.com/neovim/nvim-lspconfig/blob/master/doc/server_configurations.md#astro) and [TreeSitter](https://github.com/virchau13/tree-sitter-astro) Plugins <span style="margin: 0.25em;"><Badge variant="neutral">Community</Badge></span> - Provides syntax highlighting/treesitter parsing, intellisense, and autocompletion for Astro inside of Neovim
```

Comment:
```suggestion
- Neovim [LSP](https://github.com/neovim/nvim-lspconfig/blob/master/doc/server_configurations.md#astro) and [TreeSitter](https://github.com/virchau13/tree-sitter-astro) Plugins <span style="margin: 0.25em;"><Badge variant="neutral">Community</Badge></span> - Syntax highlighting/treesitter parsing, IntelliSense, and autocompletion for Astro inside of Neovim
```

Thanks for this! I just made a couple of small edits to match the style of the entry above it.

Reply:
Minor nitpick: auto-completion is included in the term `IntelliSense` (it's an umbrella term for many features)

Reply:
Thanks for this, @Princesseuh ! Then, I guess we should take it out of Nova as well. Would it sound "right" if we just removed autocompletion from both of those lists? Or, is there a different way you'd expect or prefer to see the description phrased?

Reply:
I personally do not have a lot of attachment to the term IntelliSense in general to be honest, it's a cool marketing term Microsoft invented notably for Visual Studio (not-Code), but I'm not actually sure people say "Look at the IntelliSense!" nowadays

I feel like just writing `code completion` might be more informative?

Reply:
Thanks, both of you! Then @TaylorBeeston , would you like to reword both Neovim and Nova together, however you see fit (make them both match each other to the extent they should), and when @Princesseuh is happy with the wording, I'm happy!

Reply:
Sure! I'll just switch it to code completion since that also works for me!